arviz/data/inference_data.py

```diff
@@ -370,6 +375,8 @@ def from_netcdf(filename, group_kwargs=None, regex=False) -> "InferenceData":
                     for key, kws in group_kwargs.items():
                         if re.search(key, group):
                             group_kws = kws
+                if HAS_H5NETCDF:
```

Comment:
setdefault can be left, just this check can be removed?

Reply:
Done!